lib/kitchen/directions/bake_notes/bake_autotitled_notes.rb

```diff
@@ -8,6 +8,14 @@ def self.v1(book:, classes:)
           next unless (note.classes & classes).any?
 
           bake_note(note: note)
+
+          next unless note.notes.present?
+
+          note.notes.each do |nested_notes|
```

Comment:
doesn't `book.notes` above already find nested notes and bake them?

I actually checked out this branch, commented out the changes, and the test still passed so I don't think this PR is necessary? :-)

Reply:
it wasn't baking the nested notes in calculus vol 3 until I did this PR